Launcher/Classes/ViewControllers/SettingsViewController.swift

```diff
@@ -60,6 +60,11 @@ class SettingsViewController: NSViewController {
         self.dismiss(true)
     }
     
+    @IBAction func clickResetToDefaults(_ sender: Any) {
```

Comment:
If there is no convention I’d go with ‘resetToDefaults’ here and in the storyboard.

Reply:
We agreed on having action name in actions and the one you proposed for outlets